index.html

```diff
@@ -0,0 +1,105 @@
+                    </h1>
+                </div>
+                <div><p class="paragraph01">Get ready to experience the vibrant beats and electrifying energy of the Caribbean as we announce 
+                    our highly anticipated World Tour musical journey through the rich cultural heritage of the Caribbean, featuring an eclectic mix of reggae, soca, calypso and more.</p></div>
+                <div><h2 class="headertime">2023.08.12(SAT) ~ 13(SUN)</h2><p class="paragraph02">@ Hasely Crawford Stadium, Port of Spain &more</p></div>
+            </section>
+        </div>
+    </div>
+    <div>
+        <section class="Program" id="program">
+            <h2>Main Program</h2>
+            <span class="program-span"></span>
+            <div class="program-list">
+                <div class="program-item"><i class="lni lni-microphone"></i><span class="prog-name">Opening Act</span><span class="prog-desc">Enjoy the performance by a surprise rising artist or band to start the night off on a high note.</span></div>
+                <div class="program-item"><i class="lni lni-music"></i><span class="prog-name">Deejay Battle</span><span class="prog-desc">Watch your favourite deejays battle on the decks with music outta caribbean and beyond.</span></div>
+                <div class="program-item"><i class="lni lni-pallet"></i><span class="prog-name">Face Paint</span><span class="prog-desc">Face painting and Face bits by the famous mona faces. Get hooked up with any face paint design.</span></div>
+                <div class="program-item"><i class="lni lni-rocket"></i></i><span class="prog-name">Final Act</span><span class="prog-desc">We shall have Final Acts of the day before close our concert graced by Sean Paul and Shaggy.</span></div>
+                <div class="program-item"><i class="lni lni-fireworks"></i><span class="prog-name">Fireworks</span><span class="prog-desc">The famous Kosh events will put up a show to watch fire works for 20 minutes straight.</span></div>
+            </div>
+            <a href="#" class="see-register">SEE THE WHOLE PROGRAM</a>
+            <a href="#" class="register">Join Caribbean Concert 2015</a>
+        </section>
+    </div>
+    <section class="featured-speakers"></section>
+    <center><div class="more-div">
+        <button class="morebtn">MORE</button>
+        <i class="lni lni-chevron-down"></i>
+    </div></center>
+    <div class="partner-container">
+        <div class="partners" id="sponsor">
+            <h2>Partner</h2>
+            <span class="span-pastevents"></span>
+            <div class="partner-list">
+                <ul>
+                    <li><img src="./images/mozilla.svg" alt=""></li>
+                    <li><img src="./images/google.png" alt=""></li>
+                    <li><img src="./images/microsoft.png" alt=""></li>
+                    <li><img src="./images/beats.png" alt=""></a></li>
+                    <li><img src="./images/airbnb.png" alt=""></a></li>
```

Comment:
- [Optional ] `Alt` atributes are important in an image, the alt attribute provides alternative information for an image if a user for some reason cannot view it. Kindly add a description for all the images in your `alt` attribute.